README.md

```diff
@@ -335,7 +335,7 @@ const PNG_LOADER = combineLoaders([
     {
         loader: 'url',
         query: {
-            limit: 100000
+            limit: 100000,
```

Comment:
Sure! Was a little behind on checking GHub but I removed the changes to the webpack config file. Should I remove the updated package.json file as well?